tools/orca/src/main.rs

```diff
@@ -97,6 +97,7 @@ fn main() -> ExitCode {
                 admin_password,
                 idm_admin_password,
                 threads,
+                Vec::new(), // for now we don't support specifying person_count_by_group from the wizard
```

Comment:
What is person_count_by_group?

Reply:
Basically it's an option that allows the user to decide how many people should be in each group instead of having them randomly assigned. This is useful when for example you want EVERYONE to be a member of `role_people_self_write_mail`. If you don't explicitly set the number of users for a given group then it will resort to the old technique we were using (that is randomly assign members). UPDATE: I've realized this is extremely dumb and works only if we assign everyone to a single group. I'll explain below what the broader issue is.

As of now, there are two factors that determine what each actor does: 

-  The first thing is obviously the `TransitionAction` that gets spit out by the `transition` function. For the `Writer` model, it will always be `WriteAttributePersonMail`
- The second factor is the groups the actor is member of. If we are not members of `role_people_self_write_mail` we'll never be able to write our email, regardless of what `TransitionAction` said. 

Before, this wasn't an issue, as all previous models derived their transitions based on what groups they belonged to, like [here](https://github.com/kanidm/kanidm/blob/master/tools/orca/src/models/basic.rs#L91), so they never tried to do something "forbidden".
Unfortunately, when we have stricter requirements, ie we need everyone to belong to a single group, the old approach won't work. We'll have actors that try to do something (i.e write their emails) and fail, as they don't have the right privileges. On the other hand, I think we could simplify it down by having just an option that says "give me the name of a group and I'll make everyone member of that group". I mean maybe in some VERY advanced *future* benchmarks one could say "alright I have 100 users, I want 80 in the mail writing group, 15 in the self read account, and so forth", but as far as I'm concerned I don't think I'll ever need such option.